src/sqlite.rs

```diff
@@ -246,6 +249,11 @@ impl TableProviderFactory for SqliteTableProviderFactory {
                 .await
                 .context(UnableToCreateTableSnafu)
                 .map_err(to_datafusion_error)?;
+        } else if !sqlite.verify_indexes_match(sqlite_conn, &indexes).await? {
+            tracing::warn!(
+                "The schema of the local copy of table '{name}' does not match the expected configuration. To correct this, you can drop the existing local copy at '{db_path}'. A new table will be automatically created with the correct schema upon the first access.",
```

Comment:
Same here on the `schema`, should only be `index`

Reply:
Updated - thank you